### PR TITLE
Migrating code from internal repository

### PR DESCRIPTION
**Pull request by shrey-garg**

### CODE REVIEW

File: lyrics/pom.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.flipkart.rome</groupId>
+    <artifactId>lyrics</artifactId>
+    <version>1.0-SNAPSHOT</version>
```
Comment (pmohankumar) [inline lyrics/pom.xml]: Can we follow semver (major, minor, patch)?

**Comment by pmohankumar** [comment]
Readme can be improved. Please add changelog, contributing.md, license file and ofc travis

**Comment by shrey-garg** [comment]
Added contributing.md and licence files, will changelog, readme and travis integration post this

File: lyrics/pom.xml (hunk 1 above)
Comment (shrey-garg) [inline lyrics/pom.xml]: done

**Pull request merged**